client/ayon_houdini/api/hda_utils.py

```diff
@@ -155,47 +172,21 @@ def _get_thumbnail(project_name: str, version_id: str, thumbnail_dir: str):
         return path
 
 
-def set_representation(node, representation_id: str):
-    file_parm = node.parm("file")
+def update_thumbnail(node):
+    if not node.evalParm("show_thumbnail"):
+        lib.remove_all_thumbnails(node)
+        return
+
+    representation_id = node.evalParm("representation")
     if not representation_id:
```

Comment:
After digging, I don't know, why the `representation_id` is equal to the lateral `"None"` after clearing the cache. 
Anyways, this works.
```suggestion
    if not representation_id or representation_id == "None":
```

Reply:
Thanks detective! This seems to hint to another issue that I'd rather fix so I'll try and reproduce and fix it.

Reply:
Fixed with https://github.com/ynput/ayon-houdini/pull/112/commits/8d85e2638c48089e8fb11e9166f043dbcc5710ce

It took me way longer than I want to admit to find that :') haha..

I did learn that I can quickly update the `hda` code continuously so I don't need to re-run houdini each time. I put e.g. this in a Houdini shelf:
```python
import hou
from ayon_houdini.api import lib, hda_utils
import importlib

# Enforce reload of AYON houdini libs
importlib.reload(lib)
importlib.reload(hda_utils)

# Reload HDAs
hou.hda.reloadAllFiles(rescan=True)

# Ensure the HDA module uses the reloaded libs
for node in hou.selectedNodes():
    hou.hda.reloadHDAModule(node.hdaModule())
```